### ISSUE TITLE
More information about incompatible mods

### ISSUE THREAD

**Issue by kianzarrin** [CitiesSkylinesMods/AutoRepair#16]
I got log message like this:
```
[ID: 1645781000] "Elektrix's Road Tools":
 - Confirmed compatible with Cities: Skylines v1.13.0 :)
 - Workshop page for this mod: https://steamcommunity.com/sharedfiles/filedetails/?id=1645781000
 - No source code/files found (yet); it might be difficult to update in future.
 - Incompatible: [ID: 1440928803] "Parallel Road Tool 1.3.0"
```
@aubergine10  explained to me : 
> it was due to an issue regarding invert flag (either being or not being set by PRT) 

@Elektrix told me on steam
>
Elektrix  [author] 11 hours ago 
kian you've even seen the code lol
Elektrix  [author] 11 hours ago 
this is compatible with parallel road tool, not sure how it wouldn't be

it would have been nice to incorporate such info in the log message why there is incompatibility. if the problem is with invert flag then I can simply invert the road after building it rather than unsubscribing from the mod. I know I can theoretically look at the comments at the steam page but sometimes there are so many comments their are buried deep under other comments. steam does not let me search the comments :(

Also would be good to let the mod owners know and encourage them to update their steam page.

**Comment by kianzarrin** [comment]
FYI Elektrix road tools had a problem with asym/1way roads. I fixed that problem for the most part but the problem persists near end nodes (i didn't bother fixing that because who wants to add  node near an end node?).

In any case @aubergine10 told me the problem is with the PRT invert flag. I will test that and get back.

**Comment by aubergine10** [comment]
I updated the records to mark them as compatible - can you try again and check it's no longer reporting incompatible?

**Comment by kianzarrin** [comment]
Sure. Thanks for updating the record. but this issue is more about giving information about how the mods are incompatible. If you agree ofcourese
This requires modification to your underlying code. you need to store dictionary of <ID, Compatibality>
where compatibility is a struct that contains status and optional message

**Comment by aubergine10** [comment]
Yes, I plan on adding notes for incompatibilities, will take some time as it's lots of typing.

I recently added feature that allows notes to be filtered based on workshop subscriptions:

![image](https://user-images.githubusercontent.com/1386719/78031020-da017180-735a-11ea-82e1-d626844695a9.png)

`NOTE` = always displayed, workshop id = only displayed if the item is subbed.

I've got code comments on many of the incompatibility defs so just need to go through and put them in to notes list keyed by relevant mod.

**Comment by kianzarrin** [comment]
I confirm that parallel road tool and Elektrix road tool are reported as compatible:
```
[ID: 1440928803] "Parallel Road Tool 1.3.0":
 - Should be compatible with current game version (if not, let us know).
 - Workshop page for this mod: https://steamcommunity.com/sharedfiles/filedetails/?id=1440928803
 - The author hasn't been active for a long time, updates unlikely.
 - Minor issues - check workshop page/comments for details.
 - Some users report major problems; check its workshop page/comments for details.
 - Source available: https://github.com/ST-Apps/CS-ParallelRoadTool
```
